SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -291,7 +296,52 @@ extension SKYChatConversationViewController {
         self.messages.append(msg)
         self.finishSendingMessage(animated: true)
 
-        // TODO: trigger finish typing event
+    }
+}
+
+
+// MARK: - Subscription
+
+extension SKYChatConversationViewController {
+
+    open func subscribeMessageChanges() {
+        if self.messageChangeHandler == nil {
```

Comment:
I don't understand why this handler has to be a property... Why don't just pass it directly to `subscribeToMessages`?